lib/response.js

```diff
@@ -229,7 +236,39 @@ res.send = function send(body) {
     this.end();
   } else {
     // respond
-    this.end(chunk, encoding);
+    if (isChunkBlob) {
+      /**
+       * @type {http.ServerResponse}
+       */
+      var _this = this;
+      /**
+       * @type {Blob}
+       */
+      var blob = chunk;
+      try {
+        var WritableStream = require('stream/web').WritableStream;
+        blob.stream().pipeTo(new WritableStream({
+          write: function (chunk) {
+            _this.write(chunk);
```

Comment:
I'm not sure why a custom write stream needs to be added, but it is missing backpressure handling, which is very important in http servers like this because the clients can stop reading if they know this is a large response, leading to a dos vector. But I would think you don't need to implement that at all, as you should be able to use `.fromWeb()`, right?

Reply:
Hi, @dougwilson, you are right here. I just modify the code in such a way that it can now handle backpressure.

Reply:
`.fromWeb()` is not yet stable so I think [stream.Readable.from(iterable[, options])](https://nodejs.org/api/stream.html#streamreadablefromiterable-options) is better to be used here.

Reply:
If i would have came here earlier to review this PR then i would have suggested stream.Readable.from(iterable) as well.